### PR TITLE
Quickstart-revamp

### PR DESCRIPTION
**Pull request by fekete-robert**
Signed-off-by: Robert Fekete <fekete77.robert@gmail.com>

### CODE REVIEW

**Comment by fekete-robert** [comment]
Hi @pepov, I've modified the loki-nginx.md file so it now has tabs for the syslog-ng/fluentd code samples. Let me know if you like it that way, then I'll delete the syslog-ng quickstart skeleton

Changing a tab now changes tabs on the entire page, so the user doesn't have to click for each step.

Screenshot 2023-07-12 at 16 40 09

File: content/docs/quickstarts/single.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+    - `Logging` to configure the logging infrastructure, like the details of the Fluent Bit and the syslog-ng or Fluentd deployment. To learn more about configuring the logging infrastructure, see {{% xref "/docs/logging-infrastructure/_index.md" %}}.
+    - `SyslogNGOutput` or `Output` to define a Loki output that receives the collected messages. To learn more, see {{% xref "/docs/configuration/output.md#syslogngoutput" %}} or {{% xref "/docs/configuration/output.md" %}}.
+    - `SyslogNGFlow` or `Flow` that processes the incoming messages and routes them to the appropriate output. To learn more, see {{% xref "/docs/configuration/flow.md#syslogngflow" %}} or {{% xref "/docs/configuration/flow.md" %}}.
+
+- Installed a simple receiver to act as the destination of the logs, and configured the the log forwarder to send the logs from the `quickstart` namespace to this destination.
+- Installed a log-generator application to generate sample log messages, and verified that the logs of this application arrive to the output.
+
+<!-- FIXME next steps/links to examples -->
```
Comment (pepov) [inline content/docs/quickstarts/single.md:323]:
```suggestion
If you have completed this guide, you have made the following changes to your cluster:

- Installed the Fluent Bit agent on every node of the cluster to collect the logs and the labels from the node.
- Installed syslog-ng or Fluentd on the cluster, to receive the logs from the Fluent Bit agents, and filter, parse, and transform them as needed, and to route the incoming logs to an output. To learn more about routing and filtering, see {{% xref "/docs/configuration/log-routing-syslog-ng.md" %}} or {{% xref "/docs/configuration/log-routing.md" %}}. - Created the following resources that configure Logging operator and the components it manages:

    - `Logging` to configure the logging infrastructure, like the details of the Fluent Bit and the syslog-ng or Fluentd deployment. To learn more about configuring the logging infrastructure, see {{% xref "/docs/logging-infrastructure/_index.md" %}}.
    - `SyslogNGOutput` or `Output` to define an http output that receives the collected messages. To learn more, see {{% xref "/docs/configuration/output.md#syslogngoutput" %}} or {{% xref "/docs/configuration/output.md" %}}.
    - `SyslogNGFlow` or `Flow` that processes the incoming messages and routes them to the appropriate output. To learn more, see {{% xref "/docs/configuration/flow.md#syslogngflow" %}} or {{% xref "/docs/configuration/flow.md" %}}.

- Installed a simple receiver to act as the destination of the logs, and configured the the log forwarder to send the logs from the `quickstart` namespace to this destination.
- Installed a log-generator application to generate sample log messages, and verified that the logs of this application arrive to the output.

<!-- FIXME next steps/links to examples -->
```

**Comment by OverOrion** [comment]
After discussing it with @pepov it might be a good idea to add a command at the start / end of the guide which allows someone to try out Fluentd after syslog-ng (or vice-versa). Maybe something like this?
```
 kubectl delete namespace logging quickstart 
```

**Comment by pepov** [comment]
It's not necessary to delete the namespaces, but it would be good to delete the resources we created, like the logging, flow and output resources. Kubernetes GC will take care of getting rid of all the dependent resources. The logging operator and the log generator can stay and we can direct the user back to the point where they had to create the logging resource.

**Comment by fekete-robert** [comment]
Something like this? "If you have already completed this quickstart with a logging forwarder and want to try the other one on the same cluster, run the following command to delete the forwarder-specific resources:

...
"

**Review by OverOrion — APPROVED** [review]
Thanks for adding it, LGTM!

**Comment by pepov** [comment]
perfect!

**Review by pepov — APPROVED** [review]

**Comment by fekete-robert** [comment]
Thanks a lot for the review!

**Pull request merged**